lib/assets/stylesheets/_accessibility.scss

```diff
@@ -4,6 +4,5 @@ a {
 }
 
 a:focus {
-  background-color: $focus-colour;
-  outline: 3px solid $focus-colour;
+  @include govuk-focused-text;
```

Comment:
@36degrees:

> I’m wary of adding the govuk-focused-text mixin to all links – I think you’re going to end up fighting it in the navigation styles. My recommendation would be to scope that just to links within the markdown-generated content and manually add focus styles to the other components that make up the page.
>
> A lot of the navigation components are going to need updating so that they behave in a similar way to the navigation elsewhere – larger click areas, but with the focus state still tight around the text